### PR TITLE
New base images and QAT DPDK testing

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

**Comment by mythi** [comment]
Testing: TBD

**Comment by mythi** [comment]
buildah is not happy using an arg variable with FROM:

```
error determining starting point for build: no FROM statement found
time="2019-08-06T19:53:35Z" level=error msg="exit status 1"
Makefile:47: recipe for target 'intel-qat-plugin' failed
make: *** [intel-qat-plugin] Error 1
```

File: deployments/qat_dpdk_app/crypto-perf/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+nameSuffix: -crypto-perf
+bases:
+- ../dpdk-test/
```
Comment (mythi) [inline deployments/qat_dpdk_app/crypto-perf/kustomization.yaml]: I slept overnight and decided to move dpdk-test, crypto-perf and compress-perf to a dir called patches/ to indicate the overlays are not usable as is.

**Comment by mythi** [comment]
> buildah is not happy using an arg variable with FROM:

buildah v1.5 should fix it. I'll update Jenkinsfile/travis.

**Comment by mythi** [comment]
@kad I've added a note about kubernetes v1.14 "dependency" with kustomize.

CI errors seem real and it's because `swupd update` returns 255 for some reason even if the update is successful. Any idea?

**Comment by ipuustin** [comment]
@mythi I noticed that too. I have a minimal `Dockerfile`:

    FROM clearlinux:latest
    RUN swupd update --no-boot-update

Building that image fails with the error:

    ...
    Update successful. System updated from version 30730 to version 30760
    The command '/bin/sh -c swupd update --no-boot-update' returned a non-zero code: 255

**Comment by ipuustin** [comment]
I went to submit a bug but there already were several: https://github.com/clearlinux/swupd-client/issues/1068

**Comment by mythi** [comment]
@kad from my side this is ready

**Comment by mythi** [comment]
@rojkov has reviewed  959f7e6 and 144b45f. They were OK.

**Comment by rojkov** [comment]
LGTM

**Pull request merged**